pages/chat/[[...roomId]].js

```diff
@@ -195,10 +204,69 @@ export default function Chat() {
                         </ServiceTable.Body>
                     </ServiceTable>
                 </details>
-                <br />
             </DefaultLayout.Sidebar>
             {roomId && (
                 <DefaultLayout.IframeWrapper>
+                    <DefaultLayout.IframeHeader>
+                        {matrix?.rooms?.get(roomId) ? (
+                            <h2>{matrix?.rooms?.get(roomId)?.name}</h2>
+                        ) : (
+                            roomId === 'new' && <h2>{t('new chat')}</h2>
+                        )}
+                        <DefaultLayout.IframeHeaderButtonWrapper>
+                            {matrix?.rooms?.get(roomId) && (
+                                <>
+                                    {' '}
+                                    <InviteUserToMatrixRoom
+                                        roomId={roomId}
+                                        trigger={
+                                            <TextButton
+                                                variant="ghost"
+                                                title={t('Invite users to {{name}}', { name: matrix.rooms.get(roomId).name })}
+                                            >
+                                                <Icon>
+                                                    <RiUserAddLine />
+                                                </Icon>
+                                            </TextButton>
+                                        }
+                                    />
+                                    <CopyToClipboard title={t('Copy chat link to clipboard')} content={'chat/' + roomId} />
+                                    <TextButton variant="ghost" title={t('Leave chat')} onClick={leaveRoom}>
+                                        <Icon>
+                                            <RiDoorOpenLine />
+                                        </Icon>
+                                    </TextButton>
+                                    <TextButton
+                                        variant="ghost"
+                                        title={t('call')}
+                                        onClick={() =>
+                                            iframe.current.contentWindow.document
+                                                .querySelector('header.mx_RoomHeader > div button:nth-child(1)')
+                                                .click()
+                                        }
+                                    >
+                                        <Icon>
+                                            <RiPhoneLine />
+                                        </Icon>
+                                    </TextButton>
+                                    <TextButton
+                                        variant="ghost"
```

Comment:
For these `<TextButton>` components there are no "variants"... so this line should be removed.

Reply:
(same goes for the other places where we're using `<TextButton>` in this PR)